packages/react/src/themes/teams/components/Menu/menuDividerStyles.ts

```diff
@@ -1,11 +1,11 @@
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { MenuDividerProps } from '../../../../components/Menu/MenuDivider'
 import { MenuVariables } from './menuVariables'
-import { getColorScheme } from '../../colors'
+import { getColorSchemeKey } from '../../colors'
 
 const menuDividerStyles: ComponentSlotStylesInput<MenuDividerProps, MenuVariables> = {
   root: ({ props: p, variables: v }): ICSSInJSStyle => {
-    const colors = getColorScheme(v.colorScheme, null, p.primary)
+    const colors = v.colorScheme[getColorSchemeKey(null, p.primary)]
```

Comment:
ouch..this doesn't look too nice, I will need to pull the branch to understand why we need to make this change; the previous `getColorScheme` was much friendlier for the theme developer

Reply:
no suggestion for now as lacking a bit of context, sorry

Reply:
I will add Teams specific interfaces to this method and will revert to using it, as this is seen clearly as something used only in the Teams theme.